WalletWasabi.Fluent/Controls/Dialog.axaml.cs

```diff
@@ -197,20 +200,53 @@ private bool ShowAlert
 		set => SetValue(ShowAlertProperty, value);
 	}
 
-	private CancellationTokenSource? CancelPointerPressedDelay { get; set; }
+	private CancellationTokenSource? CancelPointerOpenedPressedDelay { get; set; }
 
-	private void UpdateDelay(bool isDialogOpen)
+	private CancellationTokenSource? CancelPointerActivatedPressedDelay { get; set; }
+
+	protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
+	{
+		base.OnAttachedToVisualTree(e);
+
+		ApplicationHelper.MainWindowActivated.Subscribe(UpdateActivatedDelay);
```

Comment:
It should be disposed of when detaching. After changing the theme `UpdateActivatedDelay` was called many more times.

Reply:
Added